evidence/davxy/0003-davxy-2024.md

```diff
@@ -0,0 +1,74 @@
+# Evidence-0002: Retention at Rank II
+
+|                  |                      |
+| ---------------- | ---------------------|
+| **Report Date**  | 2024/11/21           |
+| **Submitted by** | Davide Galassi       |
+
+
+## Member details
+
+- Matrix username: @davxy:matrix.org
+- Polkadot address: 16Q4qkRcWd4r8196dVGNLYVfy7H86MJYJBMockPaMigFXCyv
+- Current rank: 2
+- Date of initial induction: 2024/01/08
+- Date of last report: 2024/08/20
+- Area(s) of Expertise/Interest: Consensus Protocols, Applied Cryptography
+
+
+## Reporting period
+
+- Start date: 2024/09/15
+- End date: 2024/11/15
+
+
+## Evidence
+
+### JAM
+
+During this period, my primary focus has been on the development of Parity's
+JAM prototype (*PolkaJam*). This effort included refining auxiliary crypto
+specifications and generating test vectors critical for achieving
+interoperability across all [JAM teams](https://graypaper.com/clients/).
+
+I proposed additional test vectors to advance JAM implementations'
+interoperability, specifically targeting the requirements of milestone 1 (M1).
+
+[Tracking Issue](https://github.com/w3f/jamtestvectors/issues/21)
+
+#### New Test Vector Proposals
+
+- [Binary codec update](https://github.com/w3f/jamtestvectors/pull/19)  
+- [Recent blocks history STF](https://github.com/w3f/jamtestvectors/pull/11)  
+- [Work reports STF](https://github.com/w3f/jamtestvectors/pull/20)  
+
+### Support and Collaboration
+
+I made occasional minor [contributions](https://github.com/gavofyork/graypaper/pulls?q=is%3Apr+author%3Adavxy)
+to the JAM Graypaper during this period, primarily small fixes that may not be particularly noteworthy to report
+explicitly here.
+
+Provided sporadic support to JAM implementers in the JAM public channel,
+primarily addressing topics within my areas of expertise.
+
+
+## Voting record
+
```

Comment:
|  Ranks | Activity thresholds | Agreement thresholds | Member's voting activities | Comments |
|---|---|---|---|---|
|I  |90%   |N/A   |   |  |
|II |80%   |N/A   | None  | There was nothing to vote on.  |
|III|70%   |100%  |   |  |
|IV |60%   |90%   |   |  |
|V  |50%   |80%   |   |  |
|VI |40%   |70%   |   |  |

Reply:
Recommendation is to keep the template as it is and only fill out the **rows** relevant to your rank.

Reply:
@anaelleltd https://github.com/polkadot-fellows/Evaluations/pull/85/commits/a52500d08a37805340a19e7de56bbe1943cde8a8